src/app/legacy/containers/PageHandlers/withHashChangeHandler/index.test.jsx

```diff
@@ -10,14 +10,72 @@ const Fixture = withHashChangeHandler(() => (
   </>
 ));
 
+const CyrillicFixture = withHashChangeHandler(() => (
+  <>
+    <a href="#Мирнија-сам-неко-брине-о-њој-док-је-у-школи">Go to section 1</a>
+    <section id="Мирнија-сам-неко-брине-о-њој-док-је-у-школи">
+      Section 1
```

Comment:
If using correct HTML, the text 'Section 1' should be within a tag, such as a heading, paragraph etc